contribute/internationalization.md

```diff
@@ -73,13 +76,9 @@ For components used all over the site, use just two segments:
 - `footer.update`
 - `navigation.home`
 
-### Top-level provider
+### I18next context
 
-In [AppWrapper.tsx](/public/app/AppWrapper.tsx) the app is wrapped with `I18nProvider` from `public/app/core/internationalization/index.tsx` where the Lingui instance is created with the user's preferred locale. This sets the appropriate context and allows any component from `@lingui/macro` to use the translations for the user's preferred locale.
-
-### Message format
-
-Lingui uses the [ICU MessageFormat](https://unicode-org.github.io/icu/userguide/format_parse/messages/) for the phrases in the .po catalogues. ICU has special syntax especially for describing plurals across multiple languages. For more details see the [Lingui docs](https://lingui.js.org/ref/message-format.html).
+We rely on a global i18next singleton (that lives inside the i18next) for storing the i18next config/context.
 
 ### Plain JS usage
```

Comment:
think this section could be moved under `How to add a new translation phrase`? e.g. could have 

## How to add a new translation phrase
### JSX
### Plain JS usage